internal/stage6.go

```diff
@@ -33,7 +34,7 @@ func testType1(stageHarness *test_case_harness.TestCaseHarness) error {
 		}
 	}
 
-	invalidCommands := []string{"nonexistent", "nonexistentcommand"}
+	invalidCommands := random.RandomWords(2)
```

Comment:
We can't really "guarantee" that these are invalid btw - for example, "cat" might be a random word that is returned. Not sure if this is too verbose, but how about something like `invalid_command_<random>`?

Reply:
@rohitpaulk How about `<random>_command`?

Reply:
@andy1li yep, works!

Reply:
Just went through the fixtures and what you have right now looks good btw! No need to change further